src/test/java/org/eolang/speco/SpecoTest.java

```diff
@@ -131,6 +136,28 @@ public void compilesFromEo(final String pack, @TempDir final Path temp)
         );
     }
 
+    /**
+     * Unit tests for transformations.
+     * @param pack Pack with test data
+     * @throws IOException Iff IO error
+     */
+    @Tag("fast")
+    @ParameterizedTest
+    @ClasspathSource(value = "org/eolang/speco/transformations", glob = "**.yaml")
+    public void applyTransToXmir(final String pack) throws IOException {
+        final Map<String, Object> script = new Yaml().load(pack);
+        final Train<Shift> train = new TrDefault<Shift>()
```

Comment:
@advasileva Can we inline that variable?

Reply:
@volodya-lombrozo Yes, done.